front-end/src/components/Register.js

```diff
@@ -100,7 +97,11 @@ const Register = () => {
             type="password"
             id="password"
             autoComplete="current-password"
+            onChange={(e) => {
+              setStatePassword(e.target.value);
+            }}
           />
+          <PasswordStrengthBar password={statePassword} minLength={8} />
```

Comment:
I like the password strength bar! It's a nice touch, but it seems that we can still register whatever password we want? I think we should implement the universal rules of:
1 lowercase char, 1 uppercase char, 1 num, 1 special char, and at least 8 characters long. 
https://www.thepolyglotdeveloper.com/2020/02/test-password-strength-regex-react-application/

If you want we can do this in a follow up and I can add that logic with my last PR. Lmk what you think.

Reply:
Whoops yeah that's a good call lol, working on that now

Reply:
Fixed